src/app/@ansyn/ansyn/modules/overlays/overlay-status/effects/overlay-status.effects.ts

```diff
@@ -174,11 +176,22 @@ export class OverlayStatusEffects {
 					const featurePolygons = polygons.map((polygon) => {
 						return feature(polygon);
 					});
-					const combinedResult = unifyPolygons(featurePolygons);
-					if (combinedResult.geometry.type === 'MultiPolygon') {
-						scannedArea = combinedResult.geometry;
-					} else {	// polygon
-						scannedArea = geojsonPolygonToMultiPolygon(combinedResult.geometry);
+					let combinedResult = unifyPolygons(featurePolygons);
+					const scannedAreaToPolygon = geojsonMultiPolygonToFirstPolygon(scannedArea);
+				
+					if (getPolygonIntersectionRatioWithMultiPolygon(position.extentPolygon, scannedArea) || booleanContains(scannedAreaToPolygon, position.extentPolygon)) {
```

Comment:
maybe create 2 variables that contain the values for the if instead of the long one